src/java/us/kbase/templates/module_test_python_client.vm.properties

```diff
@@ -1,124 +1,117 @@
 # -*- coding: utf-8 -*-
+import unittest
 import os
+import json
 import time
-import unittest
-from configparser import ConfigParser
 
-from ${module_name}.${module_name}Impl import ${module_name}
-from ${module_name}.${module_name}Server import MethodContext
-from ${module_name}.authclient import KBaseAuth as _KBaseAuth
+from os import environ
+try:
+    from ConfigParser import ConfigParser  # py2
+except:
+    from configparser import ConfigParser  # py3
```

Comment:
I think we should let py2 users use the existing SDK and keep this new version py3 only.